docs-article-templates/README.md

```diff
@@ -1,3 +1,4 @@
+[![Build status](https://ceapex.visualstudio.com/Engineering/_apis/build/status/Authoring/docs-article-templates%20CI)](https://ceapex.visualstudio.com/Engineering/_build/latest?definitionId=1348)
```

Comment:
@bharney-msft Should templates, preview and yaml include the branch name as well?  It looks like docs-markdown points to develop.

Reply:
Good point. I'll include the parameter for branch to be set to develop